dpnp/backend/examples/example_bs_sycl.cpp

```diff
@@ -988,35 +955,30 @@ int main(int argc, char *argv[]) {
         t[i] =  random(TL, TH);
     itt_task_end;
 
-    //------------ performance mesure ------------
-    std::vector<double> async_times;
-    for (int i = 0; i < reprtitions; ++i)
-    {
-        auto t1 = std::chrono::high_resolution_clock::now();
-        itt_task_begin(handle_async_run);
-        black_scholes(price, strike, t, RISK_FREE, VOLATILITY, call, put, SIZE, q);
-        itt_task_end;
-        auto t2 = std::chrono::high_resolution_clock::now();
-        async_times.push_back(std::chrono::duration_cast<std::chrono::duration<double>>( t2 - t1 ).count());
-    }
+    //------------ warming up ------------
+    itt_task_begin(handle_warming_up);
+    black_scholes(price, strike, t, RISK_FREE, VOLATILITY, call, put, SIZE, q);
+    itt_task_end;
 
-    std::vector<double> sync_times;
-    for (int i = 0; i < reprtitions; ++i) 
-    {
-        auto t1 = std::chrono::high_resolution_clock::now();
-        itt_task_begin(handle_sync_run);
-        black_scholes(price, strike, t, RISK_FREE, VOLATILITY, call, put, SIZE, q, true);
-        itt_task_end;
-        auto t2 = std::chrono::high_resolution_clock::now();
-        sync_times.push_back(std::chrono::duration_cast<std::chrono::duration<double>>( t2 - t1 ).count());
-    }
-    itt_task_begin(handle_init_t);
+    //------------ performance mesure ------------
+    auto t1 = std::chrono::high_resolution_clock::now();
+    itt_task_begin(handle_async_run);
+    black_scholes(price, strike, t, RISK_FREE, VOLATILITY, call, put, SIZE, q);
```

Comment:
Let's keep the loop for calling the algo and add command-line parameter to set number of repetitions. For example we can set number of repetitions equal to 1.